docs/Operators.md

```diff
@@ -1969,7 +1969,9 @@ s = np.array([1.0, 1.5]).astype(np.float32)
 bias = np.array([0, 1]).astype(np.float32)
 mean = np.array([0, 3]).astype(np.float32)
 var = np.array([1, 1.5]).astype(np.float32)
-training_mode = np.ones(1, dtype=bool)
+# using np.bool(1) while generating test data with "'bool' object has no attribute 'dtype'"
+# working around by using np.byte(1).astype(bool)
+training_mode = np.byte(1).astype(bool)
```

Comment:
`np.ones((), dtype=bool)` or `np.array(True)` would also work, this is also fine, though?

Reply:
Good to know, thanks!
Yes I tested with np.byte(1).astype(bool) and it works as well